doc/source/data/examples/batch_training.ipynb

```diff
@@ -0,0 +1,994 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# To speed things up, we’ll only use a small subset of the full dataset consisting of two last months of 2019.\n",
+    "# You can choose to use the full dataset for 2018-2019 by setting the SMOKE_TEST variable to False.\n",
+    "\n",
+    "SMOKE_TEST = True"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating Ray Dataset <a class=\"anchor\" id=\"create_ds\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[Ray Datasets](datasets) are the standard way to load and exchange data in Ray libraries and applications. We will use the [Ray Dataset APIs](dataset-api) to read the data and quickly inspect it.\n",
+    "\n",
+    "First, we will define some global variables we will use throughout the notebook, such as the list of S3 links to the files making up the dataset and the possible location IDs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "NYC Taxi using 1 file(s)!\n",
+      "s3_files: ['s3://air-example-data/ursa-labs-taxi-data/by_year/2019/06/data.parquet/ab5b9d2b8cc94be19346e260b543ec35_000000.parquet']\n",
+      "Locations: [145, 166, 152]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Define some global variables.\n",
+    "target = \"trip_duration\"\n",
+    "s3_partitions = pds.dataset(\n",
+    "    \"s3://anonymous@air-example-data/ursa-labs-taxi-data/by_year/\",\n",
+    "    partitioning=[\"year\", \"month\"],\n",
+    ")\n",
+    "s3_files = [f\"s3://{file}\" for file in s3_partitions.files]\n",
+    "\n",
+    "# Obtain all location IDs\n",
+    "location_ids = (\n",
+    "    pq.read_table(s3_files[0], columns=[\"dropoff_location_id\"])[\n",
+    "        \"dropoff_location_id\"\n",
+    "    ]\n",
+    "    .unique()\n",
+    "    .to_pylist()\n",
+    ")\n",
+    "\n",
+    "# Use smoke testing or not.\n",
+    "starting_idx = -1 if SMOKE_TEST else 0\n",
+    "sample_locations = [145, 166, 152] if SMOKE_TEST else location_ids\n",
+    "\n",
+    "# Display what data will be used.\n",
+    "s3_files = s3_files[starting_idx:]\n",
+    "print(f\"NYC Taxi using {len(s3_files)} file(s)!\")\n",
+    "print(f\"s3_files: {s3_files}\")\n",
+    "print(f\"Locations: {sample_locations}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The easiest way to create a ray dataset is to use `ray.data.read_parquet` to read parquet files in parallel onto the Ray cluster."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# # The following reads everything in the files into a ray dataset.\n",
+    "# ds = ray.data.read_parquet(s3_files)\n",
+    "# ds"
```

Comment:
should this be commented out? if so, can we say why we are not running it right now?

Reply:
I'll add a comment to the notebook and push again.